Purchases/Public/RCPurchases.m

```diff
@@ -1011,14 +1010,14 @@ - (void)receiptData:(RCReceiveReceiptDataBlock)completion {
 - (void)receiptDataWithReceiptRefreshPolicy:(RCReceiptRefreshPolicy)refreshPolicy
                                  completion:(RCReceiveReceiptDataBlock)completion {
     if (refreshPolicy == RCReceiptRefreshPolicyAlways) {
-        RCDebugLog(@"Forced receipt refresh");
+        RCDebugLog(@"%@", RCStrings.receipt.force_refreshing_receipt);
         [self refreshReceipt:completion];
         return;
     }
     NSData *receiptData = [self.receiptFetcher receiptData];
     BOOL receiptIsEmpty = receiptData == nil || receiptData.length == 0;
     if (receiptIsEmpty && refreshPolicy == RCReceiptRefreshPolicyOnlyIfEmpty) {
-        RCDebugLog(@"Receipt empty, fetching");
+        RCDebugLog(RCStrings.receipt.refreshing_empty_receipt);
```

Comment:
doesn't this cause a warning?

Reply:
@aboedo Yup it does, good catch I'll make update to any warnings I see. Would this be good to merge after fixing `@"%@"`?